### PR TITLE
Correct the menutype descriptions

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #16200

The description for the menu heading  and text sperator types are incorect

The heading is only for a parent eg Users and it cannot be used to seperate items within a menu. Thats what the text seperator is for

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_menus.ini [hunk 1]
```diff
@@ -221,9 +221,9 @@ COM_MENUS_TYPE_CONTAINER_DESC="This will create a container menu which will show
 COM_MENUS_TYPE_EXTERNAL_URL="URL"
 COM_MENUS_TYPE_EXTERNAL_URL_DESC="An external or internal URL."
 COM_MENUS_TYPE_HEADING="Menu Heading"
-COM_MENUS_TYPE_HEADING_DESC="A heading for use within menus, useful when separating menus with a separator."
+COM_MENUS_TYPE_HEADING_DESC="A heading for the parent of submenu items."
 COM_MENUS_TYPE_SEPARATOR="Text Separator"
-COM_MENUS_TYPE_SEPARATOR_DESC="A text separator."
+COM_MENUS_TYPE_SEPARATOR_DESC="A text separator, useful to seperate items within a menu."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_menus.ini]: Fix spelling `seperate`

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 035ce9a87a7df74f8337d437725263d464a09dba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18191">issues.joomla.org/tracker/joomla-cms/18191</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 035ce9a87a7df74f8337d437725263d464a09dba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18191">issues.joomla.org/tracker/joomla-cms/18191</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by infograf768** [comment]
Apparently should not be RTC
https://writingexplained.org/separate-vs-seperate-difference

**Comment by infograf768** [comment]
forget it, it was corrected.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**